wsclient/src/main/java/org/hpccsystems/ws/client/antlr/EclRecord.g4

```diff
@@ -36,14 +37,14 @@ eclfield_recref:
 ;
 
 record_def_inline:
-    OCURLY eclfield_decl ((COMMA)* eclfield_decl)* CCURLY SEMI
+    OCURLY (COMMA maxlength)? eclfield_decl ((COMMA)* eclfield_decl)* CCURLY SEMI
```

Comment:
FYI this is the official grammar for a rec def (albeit, not the inline ver)
attr := RECORD [ ( baserec ) ] [, MAXLENGTH( length ) ] [, LOCALE( locale ) ] [, PACKED ]

fields ;

[ IFBLOCK( condition )

fields ;

END; ]

[ => payload ]

END;

Reply:
Thanks. I'd never come across a record with LOCALE or PACKED on any of the clusters I regressed, and didn't know about those options.

Reply:
I'm not sure how often they show up, but figured I'd give you the full definition